nexus/test-utils/src/resource_helpers.rs

```diff
@@ -1185,7 +1185,7 @@ pub struct DiskTest<'a, N: NexusServer> {
 }
 
 impl<'a, N: NexusServer> DiskTest<'a, N> {
-    pub const DEFAULT_ZPOOL_SIZE_GIB: u32 = 10;
```

Comment:
is this just so you can divide by 4?

Reply:
It's not, it's to account for the overhead. There were a bunch of tests that didn't actually need this value to be 10 but asserted it anyway, so I removed those. There were a few left over that _did_ need this value to be a certain value, and those were left in.